swagger_spec_validator/validator20.py

```diff
@@ -174,6 +179,19 @@ def validate_spec(spec_dict, spec_url='', http_handlers=None):
     return swagger_resolver
 
 
+@lru_cache()
+def get_schema_and_resolver(schema_path):
```

Comment:
I would not cache this function as initialising a new reference resolver is pretty light weight and caching by it we might affect validation of user-provided specifications (as resource fetching would not be performed).

Considering this I would suggest to revert this and use the cached `read_file` method

Reply:
Yes it appears to not provide a performance benefit.